packages/grid/backend/backend.dockerfile

```diff
@@ -1,94 +1,43 @@
-ARG PYTHON_VERSION="3.12"
-ARG TZ="Etc/UTC"
-
-# change to USER="syftuser", UID=1000 and HOME="/home/$USER" for rootless
-ARG USER="root"
-ARG UID=0
-ARG USER_GRP=$USER:$USER
-ARG HOME="/root"
-ARG APPDIR="$HOME/app"
-
-# ==================== [BUILD STEP] Python Dev Base ==================== #
-
-FROM cgr.dev/chainguard/wolfi-base as python_dev
-
-ARG PYTHON_VERSION
-ARG TZ
-ARG USER
-ARG UID
-
-# Setup Python DEV
-RUN --mount=type=cache,target=/var/cache/apk,sharing=locked \
-    apk update && \
-    apk upgrade && \
-    apk add build-base gcc tzdata python-$PYTHON_VERSION-dev-default py$PYTHON_VERSION-pip && \
-    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
-# uncomment for creating rootless user
-# && adduser -D -u $UID $USER
+FROM cgr.dev/chainguard/wolfi-base as backend
 
-# ==================== [BUILD STEP] Install Syft Dependency ==================== #
+ARG PYTHON_VERSION="3.12"
 
-FROM python_dev as syft_deps
+RUN apk update && apk upgrade && \
+    apk add git bash python-$PYTHON_VERSION-default uv=0.1.22-r0
```

Comment:
MAYBE: we could make the uv version as an  arg variable like python version.?
recently uv release 0.1.23 , we could upgrade to that, or does it work only in this specific version?

Reply:
fixed